javaagent/build.gradle.kts

```
@@ -271,6 +273,36 @@ with(components["java"] as AdhocComponentWithVariants) {
   }
 }
 
+spdxSbom {
+  targets {
+    // Create a target to match the published jar name.
+    // This is used for the task name (spdxSbomFor<SbomName>)
+    // and output file (<sbomName>.spdx.json).
+    create("opentelemetry-javaagent") {
+      configurations.set(listOf("baseJavaagentLibs"))
+      scm {
+        uri.set("https://github.com/" + System.getenv("GITHUB_REPOSITORY"))
+        revision.set(System.getenv("GITHUB_SHA"))
+      }
+      document {
+        name.set("opentelemetry-java_opentelemetry-javaagent")
```

can/should we update here also?

```suggestion
        name.set("opentelemetry-javaagent")
```